### PR TITLE
Add v4.1 version specific cover image for PDF

### PR DESCRIPTION
**Pull request by rejahrehim**
- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Add version specific cover image for v4.1

### CODE REVIEW

**Comment by rejahrehim** [comment]
@ThunderSon Please check

File: pdf/scripts/make-pdf.sh [hunk 1]
```diff
@@ -27,11 +28,19 @@ VERSION_NUMBER=`echo $VERSION | sed 's/v//'`
 # Update build version number in pdf-config
 sed -i "s/{PDF Version}/$VERSION/g" pdf/pdf-config.json
 
-# Create the Markdown file for the cover pages
-echo "<img src=\"images/book-cover.jpg\" style=\"overflow:hidden; margin-bottom:-25px;\" />
+# Create the Markdown file for the front cover pages
+# Create the cover image with versioned image if exists else use the default with version number
+VERSIONED_COVER_IMAGE_FILE=images/book-cover-$VERSION.jpg
+if [[ -f "build/$VERSIONED_COVER_IMAGE_FILE" ]]; then
```
Comment (ThunderSon) [inline pdf/scripts/make-pdf.sh:34]: Why are you prefixing it with the `build` directory?

Comment (rejahrehim) [reply pdf/scripts/make-pdf.sh:34]: It is copied to build folder, which is referenced in the markdown.

**Comment by victoriadrake** [comment]
#234

**Pull request merged**